api/inventorypb/agents.proto

```diff
@@ -100,6 +100,12 @@ message MySQLdExporter {
   bool tls = 6;
   // Skip TLS certificate and hostname validation.
   bool tls_skip_verify = 7;
+  // Certificate Authority certificate chain.
+  string tls_ca = 15;
+  // Client certificate.
+  string tls_cert = 16;
+  // Password for decrypting tls_cert.
+  string tls_key = 17;
```

Comment:
Looks like we repeated these 3 fields over several massages. Can we extract them to a separate message? 
In this case, we will keep it more DRY and it will be more easer to access it from code as a group?

Reply:
If we will agree on this then no problem, but then it is good change this also for mongodb and RDS, because it is same case like here.